Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -93,7 +93,7 @@ try {
             $contents += "`$VerbosePreference = 'continue'"
         }
 
-        $contents += ". $PSScriptRoot\UpdatePSModulePath.ps1 $UpdatePSModulePathArgument"
+        $contents += ". `'$PSScriptRoot\UpdatePSModulePath.ps1`' $UpdatePSModulePathArgument"
```

Comment:
in case of single quote also do we need escape chars ?

Reply:
@chshrikh Not needed chaitanya. I removed the escape characters